### ISSUE TITLE
Signor processor errors

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#398]
```
======================================================================
ERROR: test_signor.test_parse_csv
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/bgyori/indra/indra/tests/test_signor.py", line 27, in test_parse_csv
    sp = SignorProcessor()
  File "/home/travis/build/bgyori/indra/indra/sources/signor.py", line 181, in __init__
    self._data = [SignorRow(*[f.strip() for f in r]) for r in data_iter]
TypeError: __new__() takes exactly 28 arguments (2 given)
```
Possibly due to the format of the CSV file having changed?

**Comment by johnbachman** [comment]
The problem appears to be that current way of retrieving the form data with a POST request:

```
res = requests.post('http://signor.uniroma2.it/download_entity.php',
                                data={'organism':'human', 'format':'csv',
                                      'submit':'Download'})
```

This used to retrieve the CSV data directly, but now it's returning an HTML page.

**Comment by johnbachman** [comment]
Looks like the problem was a required change from http to https. Fixed in 60bc87766d95.
